tools/locate_bins.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+have_command () {
+  eval "$1 --version >> /dev/null 2>&1"
+  if [ $? -ne 0 ]
+  then
+    echo "false"
+  fi
+  echo "true"
+}
+
+eval "sudo -n echo >> /dev/null 2>&1"
```

Comment:
use functions

Reply:
This needs to be executed when the script is included.  Should I just make a function and then call it at the end of the script?

Reply:
what's wrong with one-line functions? it would at least explain this line's purpose.
Generally speaking, it's cleaner to write bash scripts in a manner where you write functions and then just execute them at the end of the script or in a separate script for longer utilities. It improves readability and debugging.

Reply:
There is nothing wrong with one-line functions, I simply was unsure of what you were trying to tell me.